Luft/Settings/SettingsWindowController.m

```diff
@@ -0,0 +1,90 @@
+//
+//  SettingsWindowController.m
+//  Luft
+//
+//  Created by Emil Bogren on 15/11/15.
+//  Copyright © 2015 Hugo Tunius. All rights reserved.
+//
+
+#import <AppKit/AppKit.h>
+
+#import "SettingsWindowController.h"
+
+#import "LuftSettings.h"
+
+@interface SettingsWindowController () <NSTextFieldDelegate>
+
+@property (weak) IBOutlet NSTextField *lowerLimitTextField;
+@property (weak) IBOutlet NSStepper *lowerLimitStepper;
```

Comment:
Ah yes, I removed them. Should remove the code too, added to much complexity to my design :joy: